app/controllers/application_controller.rb

```diff
@@ -1,3 +1,4 @@
 class ApplicationController < ActionController::Base
-  protect_from_forgery with: :exception
+  # Requires api auth
+  # protect_from_forgery with: :exception
```

Comment:
We'll probably need to re-enable this and add some form of auth if we push this to prod.

Reply:
There exists ApiController, you can use that instead of this one.